### ISSUE TITLE
Status of the old Windows 10 emoji?

### ISSUE THREAD

**Issue by mindplay-dk** [microsoft/fluentui-emoji#101]
This might be the wrong place to ask, but official Windows support channels generally are a dead end for these sorts of things.

Early reactions on the net to this new set of emoji has not been great - a lot of people are asking how to remove them and restore the old ones, and I would be one of those people.

It's not only a matter of personal taste (I'm an adult and I do not like the overly colorful children's cartoon vibe) but also a matter of communication - the emoji have become part of our written language, and the sudden, drastic change makes them very difficult to decode.

Switching emoji is not as simple as changing a font - letters are abstract symbols, they don't inherently resemble anything, so most people have an easier time abstracting from precisely what the characters look like. Changing the look of an emoji is very different and much more confusing, as these symbols actually illustrate real facial expressions and real-life objects.

Emoji work by mirroring our own facial expressions - imagine you looked in the mirror one day and suddenly your face looked different. Very confusing. It's obviously not that extreme, but we do to some extend mirror our selves in these graphics, and it is very confusing to wake up to Windows 11 one day and find you have to adapt to an entirely new visual language.

I'm sure these new emoji will be very popular with kids, who are generally more adaptable than adults - but for some (many) of us, this change (any change) is unnecessary, confusing and distracting, and we don't want them.

If you had introduced these new emoji as an option, how many do you think would have chosen the old ones?

Bottom line:

Please provide an option in Windows 11 to switch back to the old Windows 10 emoji.

(It's cool that you're building these new ones in the open, but I hope that doesn't mean the old ones go unmaintained.)

**Comment by inferno986return** [comment]
I would also like the old emoji set with the thick black outlines to be open-sourced. I really like the simple yet effective style they have :-)

**Comment by bhathos** [comment]
They really were so far ahead of any other implementation — the degree of care and craftsmanship and high standards it held itself too were very special. Pre-Fluent Segoe recognized the fundamental textual nature of emoji and took it seriously enough to bring actual typographic principles to bear on their design, but they managed to do this without failing to account for those aspects of emoji which make them unique among typographical elements. They were amply expressive, full of character, but at the same time, they behaved like _glyphs._

Whoever was responsible for the previous design understood type. They understood legibility and the primacy of the reader and readability, but they also knew the tricks that keep often competing values from becoming a zero sum game. Clever depiction choices led to exaggerations in shape and silhouette that were carefully employed to optimize for the immediate and unconscious distinguishability of glyphs that would be vague stumbling blocks in lesser hands. Glyphs remained distinct even down to amazingly small sizes in many cases, with palette choices contributing similarly to the training of our eyes to register their identities without struggle. The outlining made it the only implementation that even came close to remaining _legible_ against the same diverse backgrounds that surrounding text would have no difficulty with. Case-by-case abstraction trade-offs kept the set heterogenous in precisely the right ways to enable them to not only seen but _read,_ yet they were somehow also decidedly more stylistically consistent than their peers, a surprising achievement. They didn’t _interrupt_ text because their design was informed by values that allowed them to behave as its first-class participants within text: they _were_ text.

The Fluent implementation is among the best around today, but that’s a bit more of a statement about the present state of ambient standards for craftsmanship in this domain than it is praise. I figure it’ll probably still be quite a few years till the world catches up to the level that the prior design was operating on and appreciates the ways that it raised every bar. I suspect it had the misfortune of arriving “too early” to be appreciated properly as an _technological advancement,_ not just some new “style”. I’m glad to see there are others who saw it, too, though.

I don’t comment like this with the expectation that it would lead to a change or convince anyone unconvinced of its specialness. Rather I comment to properly thank its designer(s), whoever they were, for both trying and succeeding. Work of that caliber deserves proper acknowledgment from those who noticed even if (or rather, especially if) it was before its time.

![“Most People Rejected His Message” meme that shows demonstrates the illegibility and indistinguishability of the current mouse and rat emoji and contrasts it with the immediately discernably distinct mouse and rat emoji as they were before the first Fluent Design International Day of Mourning](https://github.com/microsoft/fluentui-emoji/assets/155878287/754a6138-321e-4c40-8490-3cb351a69ad4)

**Comment by mindplay-dk** [comment]
> Whoever was responsible for the previous design understood type 

Whoever was responsible for the previous design understood design.

What is the basis of the new design? Who were they designing for? What problem was it intended to solve?

The new emoji remain unreadable to me. They don't resemble emoji on any of the other platforms or devices I use, and I can't discern their meaning. They look like the type of stickers a child would put on their lunch box - but 5 year olds are not a primary user demographic in the Windows user base, are they? So what gives? Someone should take a look at the reality of this design project, call it to a halt, and revert to the previous design.

**If there is no problem, there is no solution, and no reason for any design to be done.**

**Comment by douglasg14b** [comment]
With vision impairment I literally cannot tell what most of these emojii are at normal font sizes anymore. The details are too small that they just blur together t the pixel level.

Emojis are effectively useless to me now, I can't read them, they don't have the same meaning since they can't be distinguished from each other.

There should be an accessibility setting for these. The shading and details re lost at resolution, and make them difficult to actually make out.

**Comment by mindplay-dk** [comment]
At this point, I just want an option to switch emojis entirely.

Since the old emojis are gone, I just want the freedom to switch to a different emoji font.

But we can't even have that. It used to be possible with a registry hack, but that option is long since gone and broken, too.

Just give us (the 3 billion Android users) the option to switch to Noto emoji please! 😭
